__tests__/lib/token.js

```diff
@@ -27,13 +29,28 @@ describe( 'token tests', () => {
 
 		expect( token ).toEqual( undefined );
 	} );
+
 	test( 'should not validate expired token', () => {
 		const t = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MiwiaWF0IjoxNTE1NzExMDY5LCJleHAiOjE1MTU3OTc0Njl9.hZ-mAeoFAahak9WXqAVTOKEU7R_f1VsZfS5HqZOm-a4';
 		const token = new Token( t );
 		expect( token.valid() ).toEqual( false );
 		expect( token.expired() ).toEqual( true );
 	} );
 
+	test( 'should correctly validate token with invalid whitespace', () => {
```

Comment:
👍 

(If we have more patterns like this, we can look at https://github.com/mattphillips/jest-each for looping through and running the same test against a given test dataset.)